cli/cmd/cluster.go

```diff
@@ -743,7 +762,83 @@ var _clusterExportCmd = &cobra.Command{
 	},
 }
 
-func cmdInfo(awsClient *aws.Client, accessConfig *clusterconfig.AccessConfig, stacks clusterstate.ClusterStacks, printConfig bool, outputType flags.OutputType, disallowPrompt bool) {
+var _clusterHealthCmd = &cobra.Command{
+	Use:   "health",
+	Short: "inspect a cortex cluster's health status",
```

Comment:
nit: `inspect a cortex cluster's health status` -> `inspect the health of components in the cluster`